ios/Video/NewPlayerView.swift

```diff
@@ -0,0 +1,394 @@
+                authUrl: sourceValue.config.beacon?.authUrl.absoluteString,
+                url: sourceValue.config.beacon?.url.absoluteString,
+                headers: nil)
+            var jsConfig = JSConfig(muxData: jsMuxData, beacon: jsBeacon, convivaData: nil)
+            
+            var jsAds: RNDReactNativeDiceVideo.JSAds?
+            if let adUnits = sourceValue.ads?.adUnits.map({ adUnit -> RNDReactNativeDiceVideo.JSAds.AdUnit in
+                let queryParams = adUnit.adManifestParams?.map { param -> RNDReactNativeDiceVideo.JSAds.AdUnit.QueryParam in
+                    return RNDReactNativeDiceVideo.JSAds.AdUnit.QueryParam(key: param.key, value: param.value)
+                }
+                return RNDReactNativeDiceVideo.JSAds.AdUnit(
+                    insertionType: RNDReactNativeDiceVideo.JSAds.AdUnit.AdInsertionType(rawValue: adUnit.insertionType.rawValue)!,
+                    adFormat: RNDReactNativeDiceVideo.JSAds.AdUnit.AdFormat(rawValue: adUnit.adFormat.rawValue)!,
+                    adProvider: RNDReactNativeDiceVideo.JSAds.AdUnit.AdProvider(rawValue: adUnit.adProvider?.rawValue ?? ""),
+                    adTagUrl: adUnit.adTagUrl,
+                    adManifestParams: queryParams)
+            }) {
+                jsAds = RNDReactNativeDiceVideo.JSAds(adUnits: adUnits)
+            }
+            
+            var jsSubtitles = [RNDReactNativeDiceVideo.JSSubtitles]()
+            if let subtitles = sourceValue.subtitles {
+                for subtitle in subtitles {
+                    jsSubtitles.append(RNDReactNativeDiceVideo.JSSubtitles(language: subtitle.language, uri: subtitle.uri))
+                }
+            }
+            
+            var jsLimitedSeekableRange = RNDReactNativeDiceVideo.JSLimitedSeekableRange(start: sourceValue.limitedSeekableRange?.start, end: sourceValue.limitedSeekableRange?.end, seekToStart: sourceValue.limitedSeekableRange?.seekToStart)
+            
+            var jsNowPlaying = RNDReactNativeDiceVideo.JSNowPlaying(title: sourceValue.nowPlaying?.title, channelLogoUrl: sourceValue.nowPlaying?.channelLogoUrl, startDate: sourceValue.nowPlaying?.startDate, endDate: sourceValue.nowPlaying?.endDate)
+           
+            rndvJSSource = RNDReactNativeDiceVideo.JSSource(
+                id: sourceValue.id ?? "",
+                ima: rndvJSIma,
+                uri: sourceValue.uri,
+                drm: rndvJSDrm,
+                progressUpdateInterval: sourceValue.progressUpdateInterval ?? 6,
+                type: sourceValue.type,
+                title: sourceValue.title ?? "",
+                live: sourceValue.live,
+                partialVideoInformation: jsPartialVideoInformation,
+                isAudioOnly: sourceValue.isAudioOnly,
+                config: jsConfig,
+                imageUri: sourceValue.imageUri,
+                thumbnailsPreview: sourceValue.thumbnailsPreview,
+                resumePosition: nil,
```

Comment:
resumePositionValue should be from here

func setInitialSeek(position: Double) {
        jsProps.startAt.value = position
}